content/en/docs/concepts/storage/volume-health-monitoring.md

```diff
@@ -0,0 +1,30 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+--
+title: Volume Health Monitoring 
+content_type: concept
+---
+
+<!-- overview -->
+
+{{< glossary_tooltip text="CSI" term_id="csi" >}} volume health monitoring allows CSI Drivers to detect abnormal volume conditions from the underlying storage systems and report them as events on {{< glossary_tooltip text="PVCs" term_id="persistent-volume-claim" >}} or {{< glossary_tooltip text="Pods" term_id="pod" >}}.
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+<!-- body -->
+
+## Volume Health Monitoring
+
+The Volume Health Monitoring feature is part of Kubernetes implementation of Container Storage Interface (CSI). The Volume Health Monitoring feature is impelmeted in two components: External Health Monitor Controller and Kubelet.
+
+The External Health Monitor Controller will be deployed as a [sidecar container](https://kubernetes-csi.github.io/docs/deploying.html) together with the CSI controller driver. It triggers controller RPC to check the health condition of the CSI volumes. When an abnormal volume condition is detected on a CSI volume, an event will be reported on the PVC.
```

Comment:
I think this may be getting into too many details for end users. For end users, they need to check if their CSI driver supports this feature and that's about it right?

Reply:
> I think this may be getting into too many details for end users. 

:+1: / I agree.

Reply:
I removed the details about the sidecar.